revealjs_template.qmd

```diff
@@ -0,0 +1,117 @@
+---
+title: "<b>Title Slide<br>Max 4 Lines<br>Times New Roman (Heading)<br>40 Point</b>"
+author: "FirstName LastName<br>Title, Department<br>Endowed Chair/Additional Description"
+date: 2025-05-06
+date-format: "MMM D, YYYY"
+format: 
+  revealjs:
+    css: 
+      - _extensions/dasl/title-slide.scss
+      - _extensions/dasl/body-slide.scss
+    slide-number: true
+title-slide-attributes:
+  data-background-image: images/darkbackground.png
+  center-title-slide: false
+editor: visual
+---
+
+::: {.slide-dark style="color: white;" data-background-image="images/landacknowledgement.png" style="text-align: center;"}
```

Comment:
I'm confused about why you need `style="color: white;"` here. Is that for the font color? When I render the slides, this font color here is dark.

Reply:
Interesting - yes the intention is for the text to be made white through this tag. When I render the slides from my end, I am seeing white text, so there must be some issue in differential caching across systems.